src/connect-actions.ts

```diff
@@ -166,4 +167,12 @@ export class ConnectActions {
     await this.bluetooth.disconnect();
     await this.radioBridge.disconnect();
   };
+
+  addStatusListener = (listener: (e: ConnectionStatusEvent) => void) => {
+    this.bluetooth?.addEventListener("status", listener);
```

Comment:
Maybe this should not have ? or have ! if we can't convince typescript?

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/279/commits/b913c9c0de3c8dcf1cf01428a3c77d027209c440